NitroxClient/Debuggers/SceneDebugger.cs

```diff
@@ -79,21 +104,44 @@ private void RenderTabGameObject()
                     using (GUILayout.ScrollViewScope scroll = new GUILayout.ScrollViewScope(gameObjectScrollPos))
                     {
                         gameObjectScrollPos = scroll.scrollPosition;
-
-                        GUILayout.Label($"GameObject: {selectedObject.name}", "header");
+                        using (new GUILayout.HorizontalScope())
+                        {
+                            GUILayout.Label($"GameObject: {selectedObject.name}", "header");
+                            sendToServer = GUILayout.Toggle(sendToServer, "Send to server");
```

Comment:
Not sure if it's going to work but maybe change this to: `sendToServer = Multiplayer.Main != null && GUILayout.Toggle(sendToServer, "Send to server");`

If it doesn't then just surround it with an `if` statement to hide this toggle if Multiplayer is not active.

Because this happens on multiple places maybe just create a function for it: `RenderSendToServerToggle()`.